packages/authentication/src/__tests__/acceptance/basic-auth.acceptance.ts

```diff
@@ -3,207 +3,189 @@
-  }
-
-  function givenAuthenticatedSequence() {
-    class MySequence implements SequenceHandler {
-      constructor(
-        @inject(SequenceActions.FIND_ROUTE) protected findRoute: FindRoute,
-        @inject(SequenceActions.PARSE_PARAMS)
-        protected parseParams: ParseParams,
-        @inject(SequenceActions.INVOKE_METHOD) protected invoke: InvokeMethod,
-        @inject(SequenceActions.SEND) protected send: Send,
-        @inject(SequenceActions.REJECT) protected reject: Reject,
-        @inject(AuthenticationBindings.AUTH_ACTION)
-        protected authenticateRequest: AuthenticateFn,
-      ) {}
-
-      async handle(context: RequestContext) {
-        try {
-          const {request, response} = context;
-          const route = this.findRoute(request);
-
-          // Authenticate
-          await this.authenticateRequest(request);
-
-          // Authentication successful, proceed to invoke controller
-          const args = await this.parseParams(request, route);
-          const result = await this.invoke(route, args);
-          this.send(response, result);
-        } catch (error) {
-          this.reject(context, error);
-          return;
-        }
-      }
-    }
-    // bind user defined sequence
-    server.sequence(MySequence);
-  }
-
-  function givenProviders() {
-    class MyPassportStrategyProvider implements Provider<Strategy | undefined> {
-      constructor(
-        @inject(AuthenticationBindings.METADATA)
-        private metadata: AuthenticationMetadata,
-      ) {}
-      value(): ValueOrPromise<Strategy | undefined> {
-        if (!this.metadata) {
-          return undefined;
-        }
-        const name = this.metadata.strategy;
-        if (name === 'BasicStrategy') {
-          return new BasicStrategy(this.verify);
-        } else {
-          return Promise.reject(`The strategy ${name} is not available.`);
-        }
-      }
-      // callback method for BasicStrategy
-      verify(username: string, password: string, cb: Function) {
-        process.nextTick(() => {
-          users.find(username, password, cb);
-        });
-      }
-    }
-    server
-      .bind(AuthenticationBindings.STRATEGY)
-      .toProvider(MyPassportStrategyProvider);
-  }
-
-  function whenIMakeRequestTo(restServer: RestServer): Client {
-    return createClientForHandler(restServer.requestHandler);
-  }
-});
-
-class UserRepository {
-  constructor(
-    readonly list: {[key: string]: {profile: UserProfile; password: string}},
-  ) {}
-  find(username: string, password: string, cb: Function): void {
-    const userList = this.list;
-    function search(key: string) {
-      return userList[key].profile.id === username;
-    }
-    const found = Object.keys(userList).find(search);
-    if (!found) return cb(null, false);
-    if (userList[found].password !== password) return cb(null, false);
-    cb(null, userList[found].profile);
-  }
-}
+// import {inject, Provider, ValueOrPromise} from '@loopback/context';
```

Comment:
@jannyHou stated I should comment out these tests for the meantime. Her PR will re-write the test.

Reply:
@emonddr Could you use `describe.skip()` instead of commenting out the code?
It will be much easier for reviewers to compare the difference in the PR recovering the tests. Thanks!

See the current code change comparison: https://github.com/strongloop/loopback-next/pull/2746/commits/92b10a262233d735d022f752cf4ca82d92f38b09#diff-a5eff729d80d870cc338b162f09467be, all the code are changed, it's hard to see the real difference.